backend/src/__init__.py

```diff
@@ -12,11 +14,46 @@ def create_app(test_config=None):
       flask or pytest is running it, the app is identical.
     """
 
+    # Environment Variables
+    auth0_audience = os.getenv("AUTH0_AUDIENCE")
+    auth0_domain = os.getenv("AUTH0_DOMAIN")
+
     # Initialize application
     app = Flask(__name__)
 
+    # HTTP Security Headers
+
+    csp = {"default-src": ["'self'"], "frame-ancestors": ["'none'"]}
+
+    Talisman(
+        app,
+        force_https=False,
+        frame_options="DENY",
+        content_security_policy=csp,
+        referrer_policy="no-referrer",
+        x_xss_protection=False,
+        x_content_type_options=True,
+    )
+
+    auth0_service.initialize(auth0_domain, auth0_audience)
```

Comment:
now that we're using getenv instead of bracket references, it'd probably be worthwhile to add a guard clause above this just to make sure these two vars != `None`